src/common/routes.js

```diff
@@ -14,5 +15,6 @@ export default (
     <Route path="/:owner/:name/builds" component={Builds}/>
     <Route path="/:owner/:name/builds/:buildId" component={BuildDetails}/>
     <Route path="/login/failed" component={LoginFailed}/>
+    <Route path="/dashboard/select-repositories" component={SelectRepositories}/>
```

Comment:
Opinions on what the route should be?

- `/dashboard/katiefenn/select-repositories`
- `/dashboard/katiefenn/add-repositories`
- `/dashboard/katiefenn/add`
- `/dashboard`?